demo.js

```diff
@@ -46,3 +49,21 @@ fluid.defaults("flock.demo.oscilOscil.synth", {
         }
     }
 });
+
+
+fluid.defaults("flock.demo.oscilOscil.nexusui", {
+    gradeNames: "fluid.component",
+    listeners: {
+        onCreate: {
+            func: function(){
```

Comment:
Please move this inline function to a separate named function that is configured declaratively, and which takes the demo's environment as an argument.